src/validations/css/stylelint.js

```diff
@@ -0,0 +1,57 @@
+import Validator from '../Validator';
+
+const errorMap = {
+  'syntaxError/Unclosed block': () => ({
+    reason: 'missing-closing-curly',
+  }),
+
+  'lintRule/declaration-block-trailing-semicolon': () => ({
+    reason: 'missing-semicolon',
+  }),
+};
+
+function isSyntaxError(error) {
+  return error.name === 'CssSyntaxError';
+}
+
+class StyleLintValidator extends Validator {
+  constructor(source) {
+    super(source, 'css', errorMap);
+  }
+
+  _getRawErrors() {
+    return System.import('../linters').then(
+      ({stylelint}) => stylelint(
+        this._source
+      ).then(
+        (result) => ({result}),
+        (syntaxError) => ({syntaxError})
```

Comment:
As a corollary to the above, I believe this should just read `(syntaxError) => [syntaxError]`. Since we’re only ever going to get a single error, we should just return a one-element array.